packages/release/release-sysctl.conf

```diff
@@ -28,6 +28,10 @@ net.ipv4.ip_default_ttl = 255
 # Enable IPv4 forwarding for container networking.
 net.ipv4.conf.all.forwarding = 1
 
+# Enable IPv6 forwarding for container networking.
+net.ipv6.conf.all.forwarding = 1
+net.ipv6.conf.eth0.forwarding = 0
```

Comment:
The `eth0` setting is necessary otherwise `aws-k8s-cni` starts failing readiness probes
```
  Warning  Unhealthy  6m15s                  kubelet            Readiness probe failed: {"level":"info","ts":"2021-09-21T18:30:54.192Z","caller":"/usr/local/go/src/runtime/proc.go:203","msg":"timeout: failed to connect service \":50051\" within 1s"}
  Normal   Killing    5m55s                  kubelet            Container aws-node failed liveness probe, will be restarted
```

From asking around, it seems like the reason why we do this is because if forwarding is enabled then by default RA packets are not accepted on `eth0`. This makes it so that there will be no routes to the default gateway and `aws-node` will not be able reach the API server. If we want to keep forwarding enabled, then we need to explicitly set `accept_ra` to `2`  so it's accepted even if forwarding is enabled. (see http://strugglers.net/~andy/blog/2011/09/04/linux-ipv6-router-advertisements-and-forwarding/)

Reply:
Nice! I definitely prefer this approach; it's easier to reason about the interaction between two different settings.